src/sharding/clustermanager.js

```diff
@@ -298,15 +298,15 @@ class ClusterManager extends EventEmitter {
                         break;
 
                     case "fetchUser":
-                        this.fetchInfo(1, "fetchUser", message.id);
```

Comment:
don't worker IDs start at 1

Reply:
Not anymore :mmLol:

Reply:
i am pretty sure nodejs worker IDs start at 1

Reply:
Not sure about that tbh but this fetches the worker object through the cluster ID which is then mapped to the worker ID anyway and we know for sure cluster ID starts at 0

Reply:
you right actually this is fine, was reading the fetchInfo() code wrong